doc/fwutil/fwutil.md

```diff
@@ -454,6 +486,24 @@ Warning: <firmware_update_notification>
 New FW will be installed, continue? [y/N]: N
 Aborted!
 ```
+3. Non modular chassis platform
+```bash
+root@sonic:~# fwutil --yes --image=next --boot=cold --immediate=yes
```

Comment:
@sujinmkang I suggest to use something like: `fwutil update components fw -y --image=next -b fast`